--- linux/scripts/package/mkdebian	2025-06-28 12:57:53.650602759 +0000
+++ linux2/scripts/package/mkdebian	2025-06-29 01:37:51.597000595 +0000
@@ -77,6 +77,9 @@
 			debarch=i386
 		fi
 		;;
+	loongarch64)
+		debarch=loong64
+		;;
 	esac
 	if [ -z "$debarch" ]; then
 		debarch=$(dpkg-architecture -qDEB_HOST_ARCH)
@@ -207,9 +210,9 @@
 Build-Depends: debhelper-compat (= 12)
 Build-Depends-Arch: bc, bison, flex,
  gcc-${host_gnu} <!pkg.${sourcename}.nokernelheaders>,
- kmod, libelf-dev:native,
- libssl-dev:native, libssl-dev <!pkg.${sourcename}.nokernelheaders>,
- python3:native, rsync
+ kmod:amd64, libelf-dev:amd64,
+ libssl-dev:amd64,
+ python3:amd64, rsync:amd64
 Homepage: https://www.kernel.org/
 
 Package: $packagename-$version
